SofaKernel/framework/framework_test/helper/logging/logging_test.cpp

```diff
@@ -162,8 +162,14 @@ void f3()
 }
 
 
-TEST(LoggingTest, threadingTests_OpenIssue)
+
+TEST(LoggingTest, threadingTests)
 {
+    if(!SOFA_WITH_THREADING){
+        std::cout << "Test canceled because sofa is not compiled with SOFA_WITH_THREADING option." << std::endl ;
```

Comment:
:)
I hesitated but decided that in Logging_test I shouldn't make the assumption that msg_* is working.